studio/components/interfaces/BillingV2/Subscription/BillingBreakdown.tsx

```diff
@@ -0,0 +1,342 @@
+  const snap = useSubscriptionPageStateSnapshot()
+  const [showUsageFees, setShowUsageFees] = useState(false)
+  const { data: usage, isLoading: isLoadingUsage } = useProjectUsageQuery({ projectRef })
+  const { data: subscription, isLoading: isLoadingSubscription } = useProjectSubscriptionV2Query({
+    projectRef,
+  })
+  const {
+    data: upcomingInvoice,
+    error: upcomingInvoiceError,
+    isLoading: isLoadingUpcomingInvoice,
+  } = useUpcomingInvoiceQuery({
+    projectRef,
+  })
+  const billingCycleStart = dayjs.unix(subscription?.current_period_start ?? 0).utc()
+  const billingCycleEnd = dayjs.unix(subscription?.current_period_end ?? 0).utc()
+
+  const currentTier = subscription?.tier.supabase_prod_id
+  const isUsageBillingEnabled = subscription?.usage_billing_enabled
+  const [usageFees, fixedFees] = partition(upcomingInvoice?.lines ?? [], (item) => item.usage_based)
+  const totalUsageFees = usageFees.reduce((a, b) => a + b.amount, 0)
+
+  const hasExceededAnyLimits =
+    !isUsageBillingEnabled &&
+    Object.values(usage ?? {})
+      .map((metric) => {
+        if (typeof metric !== 'object') return false
+        if (metric.limit <= 0) return false
+        if (metric.usage > metric.limit) return true
+      })
+      .includes(true)
+
+  return (
+    <div className="grid grid-cols-12 gap-6">
+      <div className="col-span-12 lg:col-span-5">
+        <div className="sticky top-16">
+          <p className="text-base">Billing breakdown</p>
+          <p className="text-sm text-scale-1000">
+            Current billing cycle: {billingCycleStart.format('MMM DD')} -{' '}
+            {billingCycleEnd.format('MMM DD')}
+          </p>
+        </div>
+      </div>
+      {isLoadingSubscription ? (
+        <div className="col-span-12 lg:col-span-7 space-y-2">
+          <ShimmeringLoader />
+          <ShimmeringLoader className="w-3/4" />
+          <ShimmeringLoader className="w-1/2" />
+        </div>
+      ) : (
+        <div className="col-span-12 lg:col-span-7 space-y-6">
+          <p className="text-sm">Included usage summary</p>
+          <p className="text-sm text-scale-1000">
+            Your plan includes a limited amount of included usage. If the usage on your project
+            exceeds these quotas, your subscription will be charged for the extra usage.
+            Organization owners are notified each time your project approaches or exceeds the
+            included usage. Learn more
+          </p>
+
+          {hasExceededAnyLimits && (
+            <Alert
+              withIcon
+              variant="danger"
+              title="Your project's usage has exceeded its included quota"
+              actions={[
+                <Button
+                  key="upgrade-button"
+                  type="default"
+                  className="ml-4"
+                  onClick={() =>
+                    snap.setPanelKey(
+                      currentTier === PRICING_TIER_PRODUCT_IDS.FREE
```

Comment:
Let's use the `subscription.plan.id` and try to avoid using `tier` as much as possible